### DISCUSSION TITLE
Luce notturna / Emergenza

### DISCUSSION THREAD

**Discussion by nrizzano** [indomus/forum#2492] [Hardware & Firmware]
Salve, vorrei integrare delle luci notturne, preferibilmente con la doppia funzione notturna / emergenza in caso di black out.
Vorrei però qualcosa adattabile da home assistant, senza sensori crepuscolari, in modo che posso definire io gli scenari di quando attivarsi (magari basandomi su un sensore crepuscolare esterno, già connesso ad HA), definire l'intensità della luce (quindi i led devono essere dimmerabili e magari anche con la possibilità di definire la temperatura colore).
Esiste un prodotto del genere sul mercato?
Anche qualcosa di integrabile con uno shelly (o un esp32) che magari tramite contatto pulito vada ad azionare i vari comandi.
Qualcosa di smart, integrato, anche se più costoso sarebbe preferibile.

Idee?

**Comment by nrizzano** [comment]
Dimenticavo, vorrei montarli nelle comuni scatole 503 / 504, quindi qualcosa di piccolo (uno o due moduli al massimo).

**Reply by nromagna** [reply]
Ciao non ho capito bene se le vuoi esterne oppure interne...
io internamente (nel corridoio notte) ho una lampadina  philips ed un sensore sonoff di movimento impostato che si accenda al 70% dalle 18 alle 21 mentre al 10% dalle 21 alle 6 del mattino. Lo stesso sensore mi fa accendere un interruttore sonoff (che alimenta un'altra lampadina in bagno) solo dalle 23 alle 5.
Mentre per l'esterno ho risolto con un sensore movimento philips che mi accende un paio di lampadine quAa e la ad orari prestabiliti

**Comment by nrizzano** [comment]
Ciao dovrei usarli internamente, mi potresti dire quale lampada hai utilizzato?
Io vorrei fare proprio questo, collegarli ad sensore di luminosità e definire degli scenari di "dimmerazione" (passatemi il termine) diversi in funzione dell'orario in cui ci troviamo.
Per dimmerarli vorrei usare shelly dimmer 2 o qualcosa di simile.

**Reply by nromagna** [reply]
Una lampadina Philips Hue (ma può essere anche Merros o Yeelight o altro) e con HA ti crei le automazioni e regoli luminosità all'accensione.

**Comment by nrizzano** [comment]
Esiste una lampada philips hue da incasso in scatola 503?
Non riesco a trovarla.

**Reply by nromagna** [reply]
Secondo me no.
Potresti prendere una cosa del egenere ([LINK](https://www.amazon.it/Segnapasso-incasso-cassetta-scatola-applique/dp/B08FYR96KM/ref=sr_1_4?__mk_it_IT=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=2LY0MN7VI1N0W&keywords=luce%2Bscatola%2Bled%2B220v&qid=1686901190&sprefix=luce%2Bscatola%2Bled%2B220%2Caps%2C76&sr=8-4&th=1))  e poi abbinarci un sonoff o shelly (se ci sta dentro alla scatola...).

**Comment by nrizzano** [comment]
Nulla?
